azkustodata/kusto.go

```diff
@@ -12,12 +15,14 @@ import (
 	v2 "github.com/Azure/azure-kusto-go/azkustodata/internal/frames/v2"
 )
 
+type Statement = *kql.Builder
+
 // queryer provides for getting a stream of Kusto frames. Exists to allow fake Kusto streams in tests.
 type queryer interface {
 	io.Closer
 	query(ctx context.Context, db string, query Statement, options *queryOptions) (execResp, error)
 	mgmt(ctx context.Context, db string, query Statement, options *queryOptions) (execResp, error)
-	queryToJson(ctx context.Context, db string, query Statement, options *queryOptions) (string, error)
```

Comment:
Why break this APIs Name?

Reply:
1. This is a private api
2. The method no longer returns a json string - it returns a raw response object.